src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -198,39 +200,54 @@ def rebin_exposure_data(builder, risk: RiskString, data: pd.DataFrame):
 ###############################
 
 def get_relative_risk_data(builder, risk: RiskString, target: TargetString):
-    validate_relative_risk_data_source(builder, risk, target)
-    relative_risk_data = load_relative_risk_data(builder, risk, target)
+    source_type = validate_relative_risk_data_source(builder, risk, target)
+    relative_risk_data = load_relative_risk_data(builder, risk, target, source_type)
     relative_risk_data = rebin_relative_risk_data(builder, risk, relative_risk_data)
     return relative_risk_data
 
 
-def load_relative_risk_data(builder, risk: RiskString, target: TargetString):
+def load_relative_risk_data(builder, risk: RiskString, target: TargetString, source_type: str):
     distribution_type = get_distribution_type(builder, risk)
     relative_risk_source = builder.configuration[f'effect_of_{risk.name}_on_{target.name}'][target.measure]
 
-    if relative_risk_source == 'data':
+    if source_type == 'relative risk value':
```

Comment:
@collijk I left this as the branching structure rather than pulling out functions and making a dict mapping source type to function because I needed different parameters depending on source type so it ended up being a little messy with a lot of partial function calls. I can definitely go to that if you'd rather keep the dict pattern but I thought this ended up being more readable